config/date.rb

```diff
@@ -0,0 +1,5 @@
+class Date
+  def to_s
+    strftime('%Y-%m-%d')
```

Comment:
I was quite sure that it's a default behavior. Do we need it because some library redefines it somewhere? And if we need it, put it under `lib` folder not `config`.